lib/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/receive_enter_data_events.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class ReceiveEnterDataEvents extends Equatable {
+  const ReceiveEnterDataEvents();
+
+  @override
+  List<Object?> get props => [];
+}
+
+class LoadUserBalance extends ReceiveEnterDataEvents {
+  const LoadUserBalance();
+
+  @override
+  String toString() => 'LoadUserBalance';
+}
+
+class OnAmountChange extends ReceiveEnterDataEvents {
+  final String amountChanged;
+
+  const OnAmountChange({required this.amountChanged});
+
+  @override
+  String toString() => 'OnAmountChange: { OnAmountChange: $amountChanged }';
```

Comment:
String toString() => 'OnAmountChange: { OnAmountChange: $amountChanged }';
should be
String toString() => 'OnAmountChange: { amountChanged: $amountChanged }';

Reply:
fix